docs/src/variables.md

```diff
@@ -221,7 +221,7 @@ datastructures.
 
```

Comment:
I thought there were more docs to update, but maybe they're in #1597. @odow

Reply:
Yeah, they're still in the PR.

Reply:
I just fixed the remaining doctests, so assuming CI passes, #1597 is also good to merge. Depending on which one goes first, the other will have to be rebased on top with the updated docs.

Reply:
#1597 has been in progress for longer so ok with me if it gets merged first (assuming CI passes).

Reply:
Ok to merge #1597 merged first